queries.sql

```diff
@@ -1 +1,7 @@
+SELECT * FROM animals WHERE name LIKE '%mon%';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-13';
```

Comment:
- [ ] Kindly correct the `2019` date to cover all dates of the year. it should be `2019/31/12';` instead 👍